pkg/vulnerabilityreport/builder_test.go

```diff
@@ -99,7 +99,7 @@ func TestScanJobBuilder(t *testing.T) {
 	g.Expect(job).ToNot(gomega.BeNil())
 	g.Expect(job).To(gomega.Equal(&batchv1.Job{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "scan-vulnerabilityreport-fcc9884cb",
+			Name:      "scan-vulnerabilityreport-64d65c457",
```

Comment:
No reason to update the test name that I can see.

Reply:
It's the name of a scan job (not a test case) that has changed because we changed struct from `kube.Object` to `kube.ObjectRef` that is used to calculate the hash.